e2e/testcases/reconciler_finalizer_test.go

```diff
@@ -53,9 +55,18 @@ func TestReconcilerFinalizer_Orphan(t *testing.T) {
 	rootRepo := nt.RootRepos[rootSyncNN.Name]
 	deployment1NN := types.NamespacedName{Name: "helloworld-1", Namespace: testNs}
 	namespace1NN := types.NamespacedName{Name: testNs}
+	safetyNamespace1NN := types.NamespacedName{Name: rootRepo.SafetyNSName}
 
 	nt.T.Cleanup(func() {
-		cleanupSingleLevel(nt, rootSyncNN, deployment1NN, namespace1NN)
+		cleanupSyncsAndObjects(nt,
+			[]client.Object{
+				fake.RootSyncObjectV1Beta1(rootSyncNN.Name),
+			},
+			[]client.Object{
+				fake.DeploymentObject(core.Name(deployment1NN.Name), core.Namespace(deployment1NN.Namespace)),
+				fake.NamespaceObject(namespace1NN.Name),
+				fake.NamespaceObject(safetyNamespace1NN.Name),
+			})
```

Comment:
These lines are in TestReconcilerFinalizer_Foreground. Put them in a helper func.

Reply:
done